### PR TITLE
Update initialization.rst

### PR DESCRIPTION
**Pull request by xiaowuhu**
## How Has This Been Tested?
- [x] Pass the test by running: `pytest qlib/tests/test_all_pipeline.py` under upper directory of `qlib`.
- [ ] If you are adding a new feature, test on your own test scripts.

## Screenshots of Test Results (if appropriate):
1. Pipeline test:
2. Your own tests:

## Types of changes

- [ ] Fix bugs
- [ ] Add new feature
- [x] Update documentation

### CODE REVIEW

File: docs/start/initialization.rst [hunk 1]
```diff
@@ -12,12 +12,20 @@ Initialization
...
     .. code-block:: bash
     
         python scripts/get_data.py qlib_data --target_dir ~/.qlib/qlib_data/cn_data --region cn
-        
+
+if you are using Windows OS, first create a directory such as "md E:/Qlib/qlib_data", then using below command to get data (assume you clone qlib source code under e:/github/qlib):
+
+    .. code-block:: bash
+    
+        python e:/github/qlib/scripts/get_data.py qlib_data --target_dir e:/Qlib/qlib_data/cn_data --region cn
```
Comment (you-n-g) [inline docs/start/initialization.rst:27]: Qlib use  [~/.qlib/qlib_data/cn_data](https://github.com/xiaowuhu/qlib/blob/patch-1/qlib/config.py#L183) as the default data source. 
Using it as the target will be more user-friendly.

Another option is ignoring this  parameter to fallback to the [default value](https://github.com/lzh222333/qlib/blob/qlib/main/qlib/tests/data.py#L111)

Comment (xiaowuhu) [reply docs/start/initialization.rst:27]: in Windows OS, we cannot use "~/.qlib/......" as directory name

Comment (you-n-g) [reply docs/start/initialization.rst:27]: image
I think we can use this variable.

Another option is ignoring this parameter to fallback to the [default value](https://github.com/lzh222333/qlib/blob/qlib/main/qlib/tests/data.py#L111)

Comment (zhupr) [reply docs/start/initialization.rst:27]: Using `pathlib.Path().expanduser()`: https://github.com/microsoft/qlib/blob/main/qlib/tests/data.py#L42 (**python version >= 3.4**)
image

Comment (xiaowuhu) [reply docs/start/initialization.rst:27]: This works. 但是，通常用户会考虑到用一个大容量的驱动器来装载数据，而不是放在user\{alias}下面或者Home目录下(因为它会占用C盘的空间，而通常用户会考虑把数据放到D盘上，这样以后万一重新安装操作系统时，可以保留数据）。所以我个人认为用绝对路径是一个best practice.

Comment (you-n-g) [reply docs/start/initialization.rst:27]: The different settings will make subsequent steps fail.
For example:
https://qlib.readthedocs.io/en/latest/start/initialization.html#id1

I think we can tell users that the `uri`is configurable in the docs and comments.

For tutorial purposes,  running successfully is more important than robustness.

**Pull request closed without merging**